src/profile-logic/committed-ranges.js

```diff
@@ -164,10 +167,19 @@ export function stringifyCommittedRanges(
   return arrayValue.map(stringifyStartEnd).join('~');
 }
 
-export function getFormattedTimeLength(
+export function getFormattedLength(
```

Comment:
"length" doesn't mean much to me in this context. How would you feel about `getFormattedTimelineValue`?